test/face/scan/test_scan.py

```diff
@@ -0,0 +1,476 @@
+        remove(savefile)
+
+
+@pytest.mark.parametrize(
+    "_",
+    [
+        pytest.param(None, id="face.scan.rule: Cancelling loading an artifact snapshot")
+    ]
+)
+def test_scan_arti_load_nope(scantest, qtbot, mocker, _) -> None:
+    """
+    Attempt cancelling loading an artifact snapshot
+
+    :return:
+    """
+
+    """
+    Store the artifact information before trying to load
+    """
+    init_arti_type = scantest.arti_type.currentText()
+    init_arti_levl = scantest.arti_levl.currentText()
+    init_arti_rare = scantest.arti_rare.currentText()
+    init_arti_type_name = scantest.arti_type_name.text()
+    init_arti_name_main = scantest.arti_name_main.currentText()
+    init_arti_data_main = scantest.arti_data_main.text()
+    init_arti_name_a = scantest.arti_name_a.currentText()
+    init_arti_data_a = scantest.arti_data_a.text()
+    init_arti_name_b = scantest.arti_name_b.currentText()
+    init_arti_data_b = scantest.arti_data_b.text()
+    init_arti_name_c = scantest.arti_name_c.currentText()
+    init_arti_data_c = scantest.arti_data_c.text()
+    init_arti_name_d = scantest.arti_name_d.currentText()
+    init_arti_data_d = scantest.arti_data_d.text()
+
+    """
+    Perform the action of loading the artifact information
+    """
+    mocker.patch.object(file.FileHandling, "load_screenshot", return_value=(False, None, None))
+    qtbot.mouseClick(scantest.arti_cnvs_load, Qt.LeftButton)
+
+    """
+    Confirm if the user interface elements stayed the same
+    """
+    assert scantest.arti_type.currentText() == init_arti_type
+    assert scantest.arti_levl.currentText() == init_arti_levl
+    assert scantest.arti_rare.currentText() == init_arti_rare
+    assert scantest.arti_type_name.text() == init_arti_type_name
+    assert scantest.arti_name_main.currentText() == init_arti_name_main
+    assert scantest.arti_data_main.text() == init_arti_data_main
+    assert scantest.arti_name_a.currentText() == init_arti_name_a
+    assert scantest.arti_data_a.text() == init_arti_data_a
+    assert scantest.arti_name_b.currentText() == init_arti_name_b
+    assert scantest.arti_data_b.text() == init_arti_data_b
+    assert scantest.arti_name_c.currentText() == init_arti_name_c
+    assert scantest.arti_data_c.text() == init_arti_data_c
+    assert scantest.arti_name_d.currentText() == init_arti_name_d
+    assert scantest.arti_data_d.text() == init_arti_data_d
+
+
+@pytest.mark.parametrize(
+    "_",
+    [
+        pytest.param(None, id="face.scan.rule: Failing to load the artifact snapshot")
+    ]
+)
+def test_scan_arti_load_fail(scantest, qtbot, mocker, _) -> None:
+    """
+    Attempt failing to loading the artifact snapshot
+
+    :return:
+    """
+
+    """
+    Perform the action of loading the artifact snapshot
+    """
+    temp_prm = ""
+    savefile = NamedTemporaryFile(prefix="gi-loadouts-", suffix=".webp", delete=False, mode="wb")
+    savefile.write(randbytes(512*1024))
+    savefile.close()
```

Comment:
Write a comment to describe this step.

We are generating a random file as an image to make this function fail but that might not be immediately apparent to others.